__tests__/app.test.js

```diff
@@ -72,7 +72,8 @@ describe('Articles', ()=>{
             .get('/api/articles/one')
             .expect(400)
             .then((response) => {
-                expect(response.body.msg).toBe('bad request - invalid data format')
+                expect(response.body.msg).toBe("bad request - invalid SQL");
+                expect(response.body.code).toBe('22P02');
```

Comment:
We don't need to respond with 'Invalid SQL' in the error message. It isn't the user who is writing the SQL. A message of bad request will be fine. Also, you just need to respond with the 400 status code and not the SQL error code. Please remove that from the test